jwt/validate.go

```diff
@@ -89,6 +90,9 @@ func Validate(t Token, options ...ValidateOption) error {
 		case identClaim{}:
 			claim := o.Value().(claimValue)
 			claimValues[claim.name] = claim.value
+		case identClaimValidator{}:
+			pair := o.Value().(claimValidatorPair)
+			claimValidators[pair.Name] = pair.Validator
```

Comment:
Are there cases where the user might want to place multiple validators on a claim? If so this may need to be []ClaimValidator